docs/hostconfig.md

```diff
@@ -34,10 +34,11 @@ Docker bridge, 'none': no networking for this container, 'container:[name|id]':
 reuses another container network stack), 'host': use the host network stack
 inside the container.
 
-`restart_policy` is available since v1.2.0 and sets the RestartPolicy for how a
-container should or should not be restarted on exit. By default the policy is
-set to no meaning do not restart the container when it exits. The user may
-specify the restart policy as a dictionary for example:
+`restart_policy` is available since v1.2.0 and sets the container's *Restart Policy*
```

Comment:
I'd be in favor of keeping it as `RestartPolicy` since it's the name of the JSON property. Better for API users transitioning to docker-py, and still legible/understandable by all?